dts/arm/st/f4/stm32f410.dtsi

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2020 Hans Unzner
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/f4/stm32f4.dtsi>
+
+/ {
+	soc {
+		spi2: spi@40003800 {
+			compatible = "st,stm32-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40003800 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00004000>;
+			interrupts = <36 5>;
+			status = "disabled";
+			label = "SPI_2";
+		};
+
+		spi3: spi@40003c00 {
```

Comment:
Can you confirm **SPI3**, as the 0x4000 3c00 is "reserved" in the RM 0401
There ar 3 SPI I/F in the stm32f410xB : SPI1, SPI2, SPI5

Reply:
In the [datasheet](https://www.st.com/resource/en/datasheet/stm32f410rb.pdf) there is a SPI3/I2S3
![grafik](https://user-images.githubusercontent.com/67957916/103874091-08be2c80-50d1-11eb-8450-1dd8cbdbabab.png)  

But this is one year older than the RM0401, so maybe it's outdated?

Reply:
Well, I cannot tell you better than ... try

Reply:
I think SPI3 is wrong here. Everywhere else is only SPI1, SPI2, SPI5 mentioned.